gap/prop.gi

```diff
@@ -137,6 +137,10 @@ InstallMethod(IsMeetSemilatticeDigraph, "for a digraph",
 [IsDigraph],
 D -> DIGRAPHS_IsMeetSemilatticeAndMeetTable(D)[1]);
 
+InstallImmediateMethod(IsStronglyConnectedDigraph,
+HasDigraphStronglyConnectedComponents, 0,
```

Comment:
A different, or additional, approach could be to set the value of this attribute whenever `DigraphStronglyConnectedComponents`, respectively `DigraphConnectedComponents`, is computed. But I'm happy with immediate methods, and I'm happy to leave this as it is.

This might be something from a cargo cult, but I have a vague memory of it being important make it ultra explicit that a method that only applies to, say, digraphs should explicitly say that it requires a digraph. In that vain, I suggest changing this line to:
```suggestion
IsDigraph and HasDigraphStronglyConnectedComponents, 0,
```
and changing the other such line in this PR correspondingly.

I don't think there's any harm in doing it this way anyway.

Reply:
Good enough for me.  Will change!